static-site/src/components/splash/index.jsx

```diff
@@ -110,6 +110,13 @@ const Splash = () => {
           buttonText="Find out more"
           buttonLink="https://docs.nextstrain.org/en/latest/guides/communicate/narratives-intro.html"
         />
+        <Section
```

Comment:
I'd move this to 2nd (following SC2) - both due to its importance and because the abstract length is more similar to SC2's and so things line up better.

Reply:
Updated placement and reasoning in c423a517861e7b51902460d19c17c4ab46f5fa97